### ISSUE TITLE
How long did you train models on RSNA ICH and what's the evaluation protocol?

### ISSUE THREAD

**Issue by QiushiYang** [liuquande/FedIRM#2]
Thanks a lot for sharing your codes.  I have several questions about training and the results:
(1) Could you introduce how long did you train your models on RSNA ICH Detection and ISIC 2018 datasets with how many GPUs? (It seems you trained models with 100 federated rounds, i.e., 100 epochs?)
(2) What evaluation protocol did you adopt to obtain the results reported in Table 1? (Did you use the EMA model of your trainable model to test?)
(3) Curiously, did you try to adopt the consistency of weak-strong augmentation for unlabeled data (like FixMatch) on Fed-SSL to improve results? 

Many thanks ;-)

**Comment by HongzhengYang** [comment]
(1) In general, you can train 200 rounds for both datasets. We use 3 GPUs (Titan Xp).
(2) We randomly split a validation set from the training data. You can find the split code in train_main.py. For model selection, you can directly use the model of the last round or adopt the validation set. The results are similar.
(3)No. We do not employ this technique. You can try it to further improve results.

**Comment by QiushiYang** [comment]
Thanks for your kindly reply :-) I have two another questions: 
(1) Did you remember how many GPU hours you spend training the models on RSNA for 200 rounds(epochs)?
(2) In the training phase, did you utilize any pre-trained models (or just train from scratch)?
